pkg/cmd/utils.go

```diff
@@ -348,3 +350,18 @@ func PrintoutObject(objectToPrint interface{}, writer io.Writer, outputFormat st
 	}
 	return nil
 }
+
+//CheckIPPortReachable check whether IP with port is reachable with 1 min
+func CheckIPPortReachable(ip string, port string) error {
+	timeout := time.Second * 60
+	conn, err := net.DialTimeout("tcp", net.JoinHostPort(ip, port), timeout)
```

Comment:
and this works without having to dial/try multiple times within a certain amount of time?

Reply:
Yes, in my test it tries within 1 min, as I set in the timeout